node-red-container/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM nodered/node-red:latest-18
+FROM nodered/node-red:3.1.10-18
```

```suggestion
FROM nodered/node-red:3.1.11-18
```

3.1.11 is the latest - although should we consider moving to 4.0?